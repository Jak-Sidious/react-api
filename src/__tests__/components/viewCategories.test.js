import React from 'react';
import Enzyme, { mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { Form, Button } from 'semantic-ui-react';
import { notify } from 'react-notify-toast';
import { MemoryRouter } from 'react-router-dom';

import Viewcategories from '../../components/categories/viewCategories';

Enzyme.configure({ adapter: new Adapter() });

describe('<CViewcategories />', () => {
  const pathname = 'path';
  const props = {
    location: { pathname }
  }

  it('should render without crashing', () => {
    const { enzymeWrapper } = mount(
      <MemoryRouter>
        <Viewcategories {...props}/>
      </MemoryRouter>
    );
  });

});
