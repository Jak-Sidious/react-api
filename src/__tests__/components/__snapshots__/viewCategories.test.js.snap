// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Viewcategories component renders properly without crashing 1`] = `
<div
  className="mainBackground"
>
  <Navigation
    pathname="path"
  />
  <_class
    options={Object {}}
  />
  <ModalEditCat
    closeIcon={true}
    closeModal={[Function]}
    desc=""
    getCategories={[Function]}
    handleChange={[Function]}
    handleEdit={[Function]}
    name=""
    showModal={false}
  />
  <ModalCreateRecipe
    category_name=""
    closeIcon={true}
    closeModal1={[Function]}
    handleChange={[Function]}
    handleCreate={[Function]}
    redirectRecipes={[Function]}
    showModal1={false}
  />
  <br />
  <div>
    <h1>
      No Categories exist, please create some
    </h1>
    <Grid
      columns="equal"
    >
      <GridColumn
        width={7}
      />
      <GridColumn
        width={8}
      >
        <Button
          as={[Function]}
          id="creator"
          to="/catCreate"
        >
          Create a Category
        </Button>
      </GridColumn>
      <GridColumn />
    </Grid>
  </div>
</div>
`;
